lib/page/controller/page.summary/page.summary.controller.js

```diff
@@ -196,7 +196,9 @@ SummaryController.setContents = (pageInstance, userData, res, skipActions) => {
           }
         }
         const {_id, _type, name} = nameInstance
-        let question = nameInstance.legend || nameInstance.label || getInstanceTitle(_id)
```

Comment:
Barring my comments below about `getSummaryInstanceTitle`, the `nameInstance.legend || nameInstance.label || ` is surely redundant. My bad obviously since I wrote this whenever.

Moreover, I think we need to come back to this and revisit - the component schemas should probably declare which property counts as its '`title` prop

Reply:
NB. see my comments on the `let question` line below

Reply:
Thank you! Removed!